src/css.js

```diff
@@ -102,7 +102,7 @@ function boxModelAdjustment( elem, dimension, box, isBorderBox, styles, computed
 		// offsetWidth/offsetHeight is a rounded sum of content, padding, scroll gutter, and border
 		// Assuming integer scroll gutter, subtract the rest and round down
 		delta += Math.max( 0, Math.ceil(
-			elem[ "offset" + dimension[ 0 ].toUpperCase() + dimension.slice( 1 ) ] -
+			( elem[ "offset" + dimension[ 0 ].toUpperCase() + dimension.slice( 1 ) ] || 0 ) -
```

Comment:
I was anticipating the `|| 0` being farther out in this expression, and there may be problems from assuming that all SVG elements have no width or height. But that will all be resolved by unit tests covering SVG dimension getters, which this PR still needs.